### PR TITLE
Fix broken documentation build

### PR DESCRIPTION
**Pull request by greg0ire**
see https://github.com/sphinx-doc/sphinx/issues/2264
Fixes the build.

### CODE REVIEW

File: Resources/doc/cookbook/recipe_sortable_listing.rst [hunk 1]
```diff
@@ -49,13 +49,13 @@ Then we need to inject the Sortable listener. If you only have the Gedmo bundle
 
 .. code-block:: yaml
 
-	services:
-        gedmo.listener.sortable:
+    services:
+        gedmo.listener.sortable: true
```
Comment (osavchenko) [inline Resources/doc/cookbook/recipe_sortable_listing.rst]: Why we need this `true`? I doesn't see it in [gedmo documentation](https://github.com/Atlantic18/DoctrineExtensions/blob/master/doc/symfony2.md)

Comment (greg0ire) [inline Resources/doc/cookbook/recipe_sortable_listing.rst]: Good catch! It is a mistake, I added that when trying to find what exactly was invalid

**Comment by osavchenko** [comment]
Nice improvement :+1:

**Comment by greg0ire** [comment]
Thanks! And thank you for the review :)

**Comment by osavchenko** [comment]
You are welcome :smile:

**Comment by core23** [comment]
Thank you @greg0ire

**Comment by OskarStark** [comment]
good job and @core23 welcome to the club :+1: :tada: :balloon:

**Comment by greg0ire** [comment]
Yeah, nice to see the team is growing :smile:

**Comment by core23** [comment]
Thanks :)

@greg0ire Can you have a look at the other bundles? Seems that there are the same problems with sphinx

**Comment by greg0ire** [comment]
Sure, but not right now. I think someone should create an issue per failing bundle, so that people that want to have a look at it before I do can say they are working on it.

**Comment by greg0ire** [comment]
BTW, I think this should probably be backported to 2.3, and that people willing to solve this on other bundles should work from 2.3, and then 2.3 should be merged into master. Do you agree?

**Pull request merged**